source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -389,10 +391,10 @@ def wrapper(*args, **kwargs):
                     continue
                 logging.debug('for journey changing origin: {old_o} to {new_o}'
                               ', destination to {old_d} to {new_d}'
-                              .format(old_o=j.get('sections', [{}])[0].get('from').get('id'),
-                                      new_o=(g.origin_detail or {}).get('id'),
-                                      old_d=j.get('sections', [{}])[-1].get('to').get('id'),
-                                      new_d=(g.destination_detail or {}).get('id')))
+                              .format(old_o=j.get('sections', [{}])[0].get('from', {}).get('id', ''),
```

Comment:
hum that's quite strange that you needed those modification. Isn't something else broken ?